plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -127,15 +129,40 @@ public PluginManager(Config cfg) {
         skipFailedPlugins = cfg.isSkipFailedPlugins();
         hashFunction = cfg.getHashFunction();
         httpClient = null;
+        try{
+            userAgentInformation = this.getUserAgentInformation();
+        }
+        catch(UserAgentInformationException e){
+            logVerbose("Unable to get the UserAgentInformation");
+        }
     }
 
+    private String getUserAgentInformation() throws UserAgentInformationException{
+        Properties properties = new Properties();
+        ClassLoader propertiesClassLoader = this.getClass().getClassLoader();
+        try(InputStream propertiesStream = propertiesClassLoader.getResourceAsStream("Version.properties")){
+            if(propertiesStream != null){
+                properties.load(propertiesStream);
+                userAgentInformation = properties.getProperty("project.artifactId") +"/"+properties.getProperty("project.version");
```

Comment:
```suggestion
                userAgentInformation = "Jenkins plugin manager" + "/" + properties.getProperty("project.version");
```

Reply:
Is hard coding advised, I mean it was your advice right to display the artifactId, Should I commit these changes then?

Reply:
i didn't mean you had to pull it in via properties file, yes commit please..